src/tests/ftest/scrubber/performance.py

```diff
@@ -0,0 +1,73 @@
+"""
+  (C) Copyright 2018-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from ior_test_base import IorTestBase
+from ior_utils import IorMetrics
+
+
+class TestWithScrubberPerf(IorTestBase):
+    # pylint: disable=too-many-nested-blocks
+    """Basic Scrubber Test
+
+    :avocado: recursive
+    """
+
+    def test_scrubber_performance(self):
+        """JIRA ID: DAOS-7372
+        Check IOR performance with checksum enabled.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,medium
+        :avocado: tags=scrubber
+        :avocado: tags=test_scrubber_performance
```

Comment:
1. To avoid test changes that accidentally break this test, could we put this in `full_regression`?
2. I think we should add a `checksum` feature tag, and probably `ior` too
3. Please add the class name `TestWithScrubberPerf` tag. FYI I'm working on an automated check here so eventually we'll get a GHA failure/reminder about this

Reply:
Just added checksum feature. Added to full_regression.

Reply:
Done.